semantic/src/site/globals/site.overrides

```diff
@@ -617,6 +617,8 @@ a:hover {
   color: @headersInteractiveMed;
   font-size: @fontSmall;
   font-weight: @semiboldWeight;
+  flex-wrap: wrap;
+  gap: 10px;
 }
```

Comment:
wraps Stage Label and gives it a gap without affecting desktop.  'flex : 1' in label.overrides also allows the dropdown to occupy more of the space and not wrap the text inside the dropdown.
![assTabFixLabel](https://github.com/msupply-foundation/conforma-web-app/assets/17306152/235750af-9dd1-4145-a7f5-3cbfe81d9fbd)